admin/templates/configuration/tmp/configure/sql/QueryMetadata.sql

```diff
@@ -21,8 +21,8 @@ USE `qservMeta` ;
 -- Table `QCzar`
 -- -----------------------------------------------------
 CREATE TABLE IF NOT EXISTS `QCzar` (
-  `cid` INT NOT NULL AUTO_INCREMENT COMMENT 'Czar identifier',
-  `czar` CHAR(63) NOT NULL,
+  `cid` INT UNSIGNED NOT NULL AUTO_INCREMENT COMMENT 'Czar identifier',
```

Comment:
I'll remove unsigned but we are still 100% non-portable and full of mysql-isms like AUTO_INCREMENT and backticks.